DigitalLearningSolutions.Web/Controllers/VerifyEmailController.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Transactions;
+    using DigitalLearningSolutions.Data.Utilities;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.Services;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Authorize(Policy = CustomPolicies.BasicUser)]

```

Comment:
This endpoint should be unauthenticated.

Reply:
Done